src/seedu/addressbook/data/person/Birthday.java

```diff
@@ -0,0 +1,80 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a Person's birthday in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBirthday(String)}
+ */
+
+public class Birthday {
+    public static final String EXAMPLE = "09-08-1965";
+    public static final String MESSAGE_BIRTHDAY_CONSTRAINTS =
+                "Person birthdays should be separated by '-' and in the format DD-MM-YYYY";
+    public static final String MESSAGE_BIRTHDAY_MONTH_CONSTRAINTS = ", person birthday months (MM) should be within 01 to 12";
+    public static final String BIRTHDAY_VALIDATION_REGEX = "\\d{2}\\-\\d{2}\\-\\d{4}";
```

Comment:
Can just use a stricter regex that checks validity of month and day instead of checking separately later on.